### PR TITLE
Added the readme file for org.

### PR DESCRIPTION
**Pull request by AnujSsStw**
https://github.com/amplication/amplication/issues/3052#event-6966469495

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+## Hey, it's 👋 Amplication here!
+
+![Amplication](https://pbs.twimg.com/profile_banners/1294979453005770753/1656831372/1500x500)
```
Comment (yuval-hazaz) [inline profile/README.md]: please move this image to our repo

**Review by yuval-hazaz — CHANGES_REQUESTED** [review]

File: profile/README.md (hunk 1 above)
Comment (AnujSsStw) [reply profile/README.md]: can you provide a high resolution img. of the banner?

Comment (souravjain540) [reply profile/README.md]: cover v2

Here you go @AnujSsStw

**Comment by souravjain540** [comment]
@yuval-hazaz now?

**Pull request merged**